docs/MVS_parameters.rst

```diff
@@ -55,9 +55,9 @@ Common Parameters in the CSV/JSON files and in energyConversion.csv:
 
 **age_installed**: The number of years the asset has already been in operation
 
-**capex_fix**: A fixed cost to implement the asset, eg. planning costs which do not depend on the (optimized) asset capacity (€)
+**Cost_development**: A fixed cost to implement the asset, eg. planning costs which do not depend on the (optimized) asset capacity (€)
```

Comment:
```suggestion
**cost_development**: A fixed cost to implement the asset, eg. planning costs which do not depend on the (optimized) asset capacity (€)
```

Reply:
cost_development seems to start with a capital letter at every use - I've just suggested the change here

Reply:
Yeah, it is actually an intentional capital C (as in, not specific c (€/kW) but fix C (€)). But I also was unsure whether this would be regarded as a typo later on or not ;) 

Lets do a vote. Change to small letters = thumbsup.

Reply:
Ahh I see.. yes this makes sense. Maybe a short explanation so that it's not regarded as typo?

Reply:
I agree that `c` versus `C` carries a different meaning. However I don't agree that this translate into `cost` versus `Cost`, here I see `"cost"` as a normal name and not as a variable (as variable this would mean `c*o*s*t` , where `o`, `s`, and `t` are also variables :)). Therefore I vote for `cost development` and `specific cost development` for the signification of the variables.

Reply:
How about this then:
COST_DEVELOPMENT = "cost_development" (€)
SPECIFIC_COST = "specific_cost" (€/kW)
SPECIFIC_COST_OM = "specific_cost_om" (€/kW/a)
PRICE_DISPATCH = "price_dispatch" (€/kWh)